sensors/triggers/kafka/kafka.go

```diff
@@ -49,6 +49,16 @@ func NewKafkaTrigger(sensor *v1alpha1.Sensor, trigger *v1alpha1.Trigger, kafkaPr
 		var err error
 		config := sarama.NewConfig()
 
+		if kafkatrigger.Version == "" {
```

Comment:
I'd add a `kafkatrigger.GetVersion() (string, error)`

Reply:
Dot not want to import `sarama` package into models.

Reply:
good point